### PR TITLE
Fixed #31521 -- Skipped test_parsing_rfc850 test on 32-bit systems.

### PR DESCRIPTION
**Pull request by hramezani**
https://code.djangoproject.com/ticket/31521

### CODE REVIEW

File: tests/utils_tests/test_http.py [hunk 1]
```diff
@@ -325,10 +325,10 @@ def test_parsing_rfc850(self, mocked_datetime):
         utcnow_2 = datetime(2020, 11, 6, 8, 49, 37)
         utcnow_3 = datetime(2048, 11, 6, 8, 49, 37)
         tests = (
-            (utcnow_1, 'Tuesday, 31-Dec-69 08:49:37 GMT', datetime(2069, 12, 31, 8, 49, 37)),
+            (utcnow_1, 'Wednesday, 31-Dec-36 08:49:37 GMT', datetime(2036, 12, 31, 8, 49, 37)),
             (utcnow_1, 'Tuesday, 10-Nov-70 08:49:37 GMT', datetime(1970, 11, 10, 8, 49, 37)),
             (utcnow_1, 'Sunday, 06-Nov-94 08:49:37 GMT', datetime(1994, 11, 6, 8, 49, 37)),
-            (utcnow_2, 'Wednesday, 31-Dec-70 08:49:37 GMT', datetime(2070, 12, 31, 8, 49, 37)),
```
Comment (felixxm) [inline tests/utils_tests/test_http.py:331]: That's not a proper fix, because it doesn't test the same cases, see RFC-850.

Comment (hramezani) [reply tests/utils_tests/test_http.py:331]: As we will face an error with dates with year greater than 2039, I've decided to move tests to 100 years ago.
Don't know is it good or not?

Comment (hramezani) [reply tests/utils_tests/test_http.py:331]: They failed on windows.
I will try to find a better fix.

**Comment by felixxm** [comment]
I don't think it's worth to spend more time on this. We can skip this test on 32-bit platforms.

**Comment by hramezani** [comment]
> I don't think it's worth to spend more time on this. We can skip this test on 32-bit platforms.

I've changed it and skipped that specific test for `32-bit platforms`

**Comment by felixxm** [comment]
@hramezani Thanks :+1:

**Pull request merged**